public/v2/apps/jitsi.json

```diff
@@ -0,0 +1,255 @@
+{
+    "captainVersion": "2",
+    "documentation": "https://github.com/jitsi/docker-jitsi-meet",
+    "displayName": "Jitsi",
+    "description": "Free open-source video conferencing for web & mobile",
+    "dockerCompose": {
+      "version": "3",
+      "services": {
+        "$$cap_appname-web": {
+          "image": "jitsi/web:$$cap_tag_version",
+          "restart": "unless-stopped",
+          "ports": ["$$cap_https_port:443", "$$cap_http_port:80"],
```

Comment:
we shouldn't map the http(s) ports. Why don't we use the `containerHttpPort` ?

Reply:
How does it work? I have been trying and I couldn't make it work with that property

Reply:
Port 443 without certs doesn't provide a valid HTTPS connection. You probably have to use port `80`. So simply remove this `ports` section. You don't even need to specify `containerHttpPort` as the default value is already 80. So the network traffic will be
```
User === HTTPS ===>> nginx (CapRover) : port 433 === http ===>> jitsi : port 80
```